virttest/utils_disk.py

```diff
@@ -600,6 +601,90 @@ def create_filesystem(session, partition_name, fstype, ostype, timeout=360):
         create_filesyetem_linux(session, partition_name, fstype, timeout)
 
 
+def resize_xfs_filesystem(session, partition, fstype='xfs', size=None):
```

Comment:
Hi lolyu, 
thanks for your review, session.cmd also can catch nozero cmd status, it will raise ShellCmdError directly, and will return the output of cmd, I think it's enough for use to know what happen when error occur~ so no use session.cmd_status() here.